packages/tooling/node/src/process.ts

```
@@ -256,6 +256,16 @@ const logScriptStart = (
   console.log("")
 }
 
+const logSignerInfo = (tooling: Tooling) => {
+  const ownerAddress = tooling.loadedEd25519KeyPair.toSuiAddress()
+  const buyerAddress = tooling.loadedBuyerEd25519KeyPair.toSuiAddress()
+  const hasSeparateBuyer = ownerAddress !== buyerAddress
+  logKeyValueBlue("Signer (owner)")(ownerAddress)
+  if (hasSeparateBuyer) logKeyValueBlue("Signer (buyer)")(buyerAddress)
+  else logWarning("SUI_BUYER_ACCOUNT_* not set — buyer scripts will sign as owner. Set buyer vars to separate roles.")
```

ESLint

[error] Replace `·logWarning("SUI_BUYER_ACCOUNT_*·not·set·—·buyer·scripts·will·sign·as·owner.·Set·buyer·vars·to·separate·roles."` with `⏎····logWarning(⏎······"SUI_BUYER_ACCOUNT_*·not·set·—·buyer·scripts·will·sign·as·owner.·Set·buyer·vars·to·separate·roles."⏎····`

(prettier/prettier)